packages/plugins/plugin-navtree/src/components/NavTree.tsx

```diff
@@ -86,7 +83,7 @@ const NavTreeColumns = ({
 
   const ActionRoot = popoverAnchorId === `dxos.org/ui/${NAV_TREE_ITEM}/${node.id}` ? Popover.Anchor : Fragment;
 
-  // TODO(thure): Ideally this should not be necessary.
+  // TODO(thure): Ideally this should not be necessary. (Explain why it's required?)
```

Comment:
We've not found another way to load the graph at the right time than to load the children of any node which is renderer so that if it's expanded it's children are available. Ideally this would not be done within React as an useEffect hook but everything else to this point has proven either too complicated or didn't work.